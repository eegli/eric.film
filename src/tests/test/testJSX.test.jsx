import { shallow } from '@/src/tests/testUtils';

const Page = () => <div>Test</div>;

describe('Test - JSX', () => {
  it('renders test JSX component', () => {
    const wrapper = shallow(<Page />);
    expect(wrapper).toMatchSnapshot();
  });
});
