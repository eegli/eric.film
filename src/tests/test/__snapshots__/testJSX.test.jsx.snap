// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test - JSX renders test JSX component 1`] = `
<div>
  Test
</div>
`;
